src/_guides/language/sound-problems.md

```diff
@@ -79,13 +79,30 @@ How you troubleshoot strong mode depends on whether you are running
 
 #### Command line analyzer
 
-If you are running `dartanalyzer --strong` and you don't see expected
-strong mode errors, be sure that you didn't disable strong mode in an
-analysis options file in the same directory where you run the analyzer.
-If you do, the analysis options file overrides your command line flag.
+If you are running dartanalyzer from the command line and you don't see
+expected strong mode errors, make sure of the following:
+
+<ul markdown="1">
+<li markdown="1">
+  If you're using an [analysis
+  options](/guides/language/analysis-options#the-analysis-options-file) file,
+  make sure you've specified `strong mode: true` correctly.
+  For more information, see [Specifying strong
+  mode.](/guides/language/analysis-options#specifying-strong-mode)</li>
+
+<li markdown="1">
+  If you aren't using an [analysis
```

Comment:
This bit of help sounds incomplete.

It seems like they might want to do this even if they ARE using an analysis options file, to check whether they're getting the same results.